deploy/docker/dev.sh

```diff
@@ -3,4 +3,6 @@
 /bin/sh /code/deploy/docker/common.sh
 /code/deploy/docker/wait-for-it.sh db:5432 -- invoke restore_from_dump
 /code/deploy/docker/wait-for-it.sh solr:8983 -- python3 manage.py rebuild_index --noinput
+/code/deploy/docker/wait-for-it.sh redis:6379 -- redis-server
+cron
```

Comment:
You shouldn't need to run cron manually - the base Docker image has cron running already. Just need to add the script to be run by cron into `/etc/cron.daily/` which is already being done.